packages/babel-plugin-proposal-decorators/test/fixtures/2021-12-accessors--to-es2015/static-private/output.js

```diff
@@ -1,17 +1,9 @@
 var _initStatic, _init_a, _get_a, _set_a, _init_b, _get_b, _set_b, _Foo;
 const dec = () => {};
-var _a = /*#__PURE__*/new WeakMap();
-var _b = /*#__PURE__*/new WeakMap();
```

Comment:
Now it's always named based on the first property, can you make it based on the class?

Reply:
I'll do this in a separate PR, mostly because it makes it very difficult to go through the diff and see what actually changed.